code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -1,285 +1,646 @@
+	switch(phase)
+		if(CLOCK_DEINIT_LOGOFF_PHASE)
+			message = "Shutting off,, [U.real_name]. " + message
+	to_chat(U, "<span class='notice'>[message]</span>")
+	playsound(U, 'sound/items/deconstruct.ogg', 10, TRUE)
+
+	if (phase < CLOCK_COMPLETE_PHASE)
+		addtimer(CALLBACK(src, .proc/deinitialize, delay, U, phase + 1), delay)
+	else
+		unlock_suit()
+		s_initialized = FALSE
+		s_busy = FALSE
+
+#undef CLOCK_LOCK_PHASE
+#undef CLOCK_COMPLETE_PHASE
+#undef CLOCK_DEINIT_LOGOFF_PHASE
+
+// Ratv' Maga martial art (I know, Awful joke)
+
+/obj/item/book/granter/martial/justiciar_hand
+	martial = /datum/martial_art/justiciar_hand
+	name = "Ancient Brass Book"
+	martialname = "Justiciar Hand"
+	desc = "This book seems to depict an ancient War Machine, labeled 'The Clockwork Justiciar'."
+	greet = "<span class='sciradio'>You have learned Ratvar's martial art, Justiciar's Hand!</span>"
+	icon = 'icons/obj/library_fulp.dmi'
+	icon_state ="book_ratvar"
+	remarks = list("Wait, these guys made a brass wall and still lost?","For the Justiciar!", "The Ancient God...", "Nar'Sie was kinda mean...", "This space station will be met with its acolytes...", "His remains are at Lavaland?")
+
+#define REPULSIVE_FORCE "DGH"
+#define BRAIN_TAZE "GGH"
+#define FIREY_TOUCH "HG"
+#define TIMESTOP_CRUSH "HHD"
+
+/datum/martial_art/justiciar_hand
+	name = "Justiciar's Hand"
+	id = MARTIALART_JUSTICIARHAND
+	allow_temp_override = FALSE
+	help_verb = /mob/living/carbon/human/proc/justiciar_hand_help
+
+/datum/martial_art/justiciar_hand/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	add_to_streak("G",D)
+	if(check_streak(A,D))
+		return TRUE
+	log_combat(A, D, "grabbed (Justiciar's Hand)")
+	return ..()
+
+/datum/martial_art/justiciar_hand/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	add_to_streak("H",D)
+	if(check_streak(A,D))
+		return TRUE
+	var/obj/item/bodypart/affecting = D.get_bodypart(ran_zone(A.zone_selected))
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	var/atk_verb = pick("burn", "incinerate", "strike", "cremate")
+	D.visible_message("<span class='danger'>[A] [atk_verb]s [D]!</span>", \
+					"<span class='userdanger'>[A] [atk_verb]s you!</span>", null, null, A)
+	to_chat(A, "<span class='danger'>You [atk_verb] [D]!</span>")
+	D.apply_damage(rand(10,15), BURN, affecting, wound_bonus = CANT_WOUND)
+	playsound(get_turf(D), 'sound/weapons/punch1.ogg', 25, TRUE, -1)
+	log_combat(A, D, "punched (Justiciar's Hand)")
+	return TRUE
+
+/datum/martial_art/justiciar_hand/disarm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	add_to_streak("D",D)
+	if(check_streak(A,D))
+		return TRUE
+	log_combat(A, D, "disarmed (Justiciar's Hand)")
+	return ..()
+
+
+/datum/martial_art/justiciar_hand/proc/check_streak(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(!can_use(A))
+		return FALSE
+	if(findtext(streak,REPULSIVE_FORCE))
+		streak = ""
+		Repulse(A,D)
+		return TRUE
+	if(findtext(streak,FIREY_TOUCH))
```

Comment:
*fiery

you'll need to replace every mention of "firey" with "fiery", as the latter is the correct spelling of the word